examples/SimpleQA/SimpleQA.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluating a Question-Answering system\n",
```

Comment:
Can we keep "SimpleQA" in the title and link to [this blog](https://openai.com/index/introducing-simpleqa/) in the intro paragraph? Want to clearly attribute credit to the OpenAI folks' for contributing the dataset and mention how this is how you can reproduce it in Braintrust.